Makefile

```diff
@@ -44,11 +44,12 @@ clean:
 	@rm -f $$(which amp)
 
 test:
-	@go test -v $(REPO)/data/storage/etcd
-	@go test -v $(REPO)/data/influx
-	@go test -v $(REPO)/api/rpc/project
-	@go test -v $(REPO)/api/rpc/service
-	@go test -v $(REPO)/api/rpc/stat
+#	@go test -v $(REPO)/data/storage/etcd
```

Comment:
Don't want to merge a PR with tests commented out. Better to tell the PR review in your description to cd to `api/rpc/logs` and run `go test`.